src/main/java/com/google/maps/GeoApiContext.java

```diff
@@ -339,16 +339,30 @@ public Builder requestHandlerBuilder(RequestHandler.Builder builder) {
     }
 
     /**
-     * Overrides the base URL of the API endpoint. Useful only for testing.
+     * Overrides the base URL of the API endpoint. Useful for testing or certain international
+     * usage scenarios, such as querying the API from China.
```

Comment:
Can you please delete the reference to China in this doc comment? Thanks!

Reply:
Done!